cmd/cluster/create.go

```diff
@@ -85,6 +88,9 @@ func NewCreateCommand() *cobra.Command {
 		Annotations:        []string{},
 		NetworkType:        string(v1alpha1.OpenShiftSDN),
 		FIPS:               false,
+		RootVolumeType:     "gp2",
```

Comment:
just a note for future, at some point we'll need to discriminate these flags per platform